rocketmq-connect/rocketmq-connect-runtime/src/main/java/org/apache/rocketmq/connect/runtime/service/ConfigManagementServiceImpl.java

```diff
@@ -169,6 +230,7 @@ public String putConnectorConfig(String connectorName, ConnectKeyValue configs)
         if (errorMessage != null && errorMessage.length() > 0) {
             return errorMessage;
         }
+        // TODO is this the problem ? Put is executed after remove ?
```

Comment:
Some TODO are left in code, try to reduce them as much ap

Reply:
Fixed